src/main/java/gregtech/api/unification/material/materials/FirstDegreeMaterials.java

```diff
@@ -0,0 +1,1227 @@
+                .build();
+
+        Lepidolite = new Material.Builder(382, "lepidolite")
+                .dust().ore(5, 1)
+                .color(0xF0328C).iconSet(FINE)
+                .components(Potassium, 1, Lithium, 3, Aluminium, 4, Fluorine, 2, Oxygen, 10)
+                .build();
+
+        Glauconite = new Material.Builder(383, "glauconite")
+                .dust().ore()
+                .color(0x82B43C)
+                .components(Potassium, 1, Magnesium, 2, Aluminium, 4, Hydrogen, 2, Oxygen, 12)
+                .build();
+
+        GlauconiteSand = new Material.Builder(384, "glauconite_sand")
+                .dust()
+                .color(0x82B43C).iconSet(SAND)
+                .components(Potassium, 1, Magnesium, 2, Aluminium, 4, Hydrogen, 2, Oxygen, 12)
+                .build();
+
+        Malachite = new Material.Builder(385, "malachite")
+                .dust().ore()
+                .color(0x055F05)
+                .components(Copper, 2, Carbon, 1, Hydrogen, 2, Oxygen, 5)
+                .build();
+
+        Mica = new Material.Builder(386, "mica")
+                .dust()
+                .color(0xC3C3CD).iconSet(FINE)
+                .components(Potassium, 1, Aluminium, 3, Silicon, 3, Fluorine, 2, Oxygen, 10)
+                .build();
+
+        Barite = new Material.Builder(387, "barite")
+                .dust().ore()
+                .color(0xE6EBEB)
+                .components(Barium, 1, Sulfur, 1, Oxygen, 4)
+                .build();
+
+        Alunite = new Material.Builder(388, "alunite")
+                .dust()
+                .color(0xE1B441).iconSet(METALLIC)
+                .components(Potassium, 1, Aluminium, 3, Silicon, 2, Hydrogen, 6, Oxygen, 14)
+                .build();
+
+        Dolomite = new Material.Builder(389, "dolomite")
+                .dust(1)
+                .color(0xE1CDCD).iconSet(FLINT)
+                .components(Calcium, 1, Magnesium, 1, Carbon, 2, Oxygen, 6)
+                .build();
+
+        Wollastonite = new Material.Builder(390, "wollastonite")
+                .dust()
+                .color(0xF0F0F0)
+                .components(Calcium, 1, Silicon, 1, Oxygen, 3)
+                .build();
+
+        Kaolinite = new Material.Builder(391, "kaolinite")
+                .dust()
+                .color(0xF3EBEB)
+                .components(Aluminium, 2, Silicon, 2, Hydrogen, 4, Oxygen, 9)
+                .build();
+
+        Talc = new Material.Builder(392, "talc")
+                .dust().ore()
+                .color(0x5AB45A).iconSet(FINE)
+                .components(Magnesium, 3, Silicon, 4, Hydrogen, 2, Oxygen, 12)
+                .build();
+
+        Soapstone = new Material.Builder(393, "soapstone")
+                .dust(1).ore(3, 1)
+                .color(0x5F915F)
+                .components(Magnesium, 3, Silicon, 4, Hydrogen, 2, Oxygen, 12)
+                .build();
+
+        Kyanite = new Material.Builder(394, "kyanite")
+                .dust()
```

Comment:
Was this supposed to have an ore block? I forget

Reply:
Yes, but it did not originally in CE so I just haven't added it. There are a lot of materials like this